Sources/Datadog/RUM/Instrumentation/Actions/SwiftUI/SwiftUIActionModifier.swift

```diff
@@ -51,6 +51,50 @@ public extension SwiftUI.View {
     }
 }
 
+#else
+
+/// `SwiftUI.ViewModifier` for RUM which invoke `addUserAction` from the
+/// global RUM Monitor when the modified view receives a click from the remote control.
+@available(tvOS 14, *)
+internal struct RUMClickActionModifier: SwiftUI.ViewModifier {
+    /// The minimum duration in seconds of the long press that must elapse before the gesture succeeds.
+    let minimumDuration: TimeInterval
+
+    /// Action Name used for RUM Explorer.
+    let name: String
+
+    /// Custom attributes to attach to the Action.
+    let attributes: [AttributeKey: AttributeValue]
+
+    func body(content: Content) -> some View {
+        content.simultaneousGesture(
+            LongPressGesture(minimumDuration: minimumDuration).onEnded { _ in
+                Global.rum.addUserAction(type: .click, name: name, attributes: attributes)
+            }
+        )
+    }
```

Comment:
Ok, this **does not work** actually. `simultaneousGesture` does not seem to work properly!